### PR TITLE
style.css

### PR DESCRIPTION
**Pull request by bolaram**
Removing underline in footer menus . I think it looks good :)

### CODE REVIEW

**Comment by bolaram** [comment]
@acabunoc  will you merge this please ?

**Comment by acabunoc** [comment]
Thanks for the pr! I'm at a conference right now, I'll review next week :)

**Comment by bolaram** [comment]
Ok thanks .. @acabunoc

File: public/css/style.css [hunk 1]
```diff
@@ -545,10 +546,12 @@ footer ul.footer-links li a {
   text-transform: uppercase;
   color: #FFF;
   font-size:0.8em;
+  text-decoration:none;
 }
 
 footer ul a:hover, footer ul.footer-links li a:hover {
   color:f47e3c;
+  text-decoration:none;
```
Comment (acabunoc) [inline public/css/style.css]: I believe this is the only line needed for this change to show up. Can you remove the other changes?

**Comment by bolaram** [comment]
@acabunoc done!

**Comment by acabunoc** [comment]
Thanks @bolaram! :smile:

**Comment by bolaram** [comment]
Thanks @acabunoc :)

**Pull request merged**